src/components/GameCards/GameCardGrid.vue

```diff
@@ -19,12 +19,22 @@
           @click.native="openDetails"
         />
 
-        <progress
-          v-if="gameProgress"
-          max="100"
-          :value="gameProgress"
-          @click="openDetails"
-        />
+        <div class="game-progress">
```

Comment:
this would be a good candidate to make it a reusable component, maybe we can rename current `GameProgress` to something like `GameProgressModal` and make a new `GameProgress` component to use here